etc/config/crystal.amazon.properties

```diff
@@ -10,9 +10,13 @@ group.crystal.supportsExecute=true
 group.crystal.compilerType=crystal
 group.crystal.objdumper=/opt/compiler-explorer/gcc-snapshot/bin/objdump
 
-compiler.crystal131.semver=1.3.1
```

Comment:
Please leave the old compilers in. We don't remove compilers (else folks who've bookmarked things will get errors).

Reply:
I'm so sorry- @RubenRBS has reminded me and pointed out that you're aliasing here :blush: All's well; thanks!